mosviz/loaders/loader_selection.py

```diff
@@ -119,6 +121,8 @@ def __init__(self, parent=None, data=None):
         # sophisticated auto-testing.
 
         for column in self.columns:
+            if column['components'] == []:
```

Comment:
Splitting hairs about performance, but you could avoid instantiating a new list by relying on the falsiness of empty objects: `if not column['components']`.